bigquery/google/cloud/bigquery/job.py

```diff
@@ -742,7 +742,10 @@ def from_api_repr(cls, resource, client):
         dataset = Dataset(dest_config['datasetId'], client)
         destination = Table(dest_config['tableId'], dataset)
         sources = []
-        for source_config in config['sourceTables']:
+        source_configs = config.get('sourceTables')
+        if source_configs is None:
+            source_configs = [config['sourceTable']]
```

Comment:
Server docs say "pick one" between `sourceTable` and `sourceTables`:  one cannot create a "copy" job without specifying one of them.